2_segmenting_data/Segmented_Images/D6_3_cyto-segmented.tsv

```diff
@@ -0,0 +1,2 @@
+Cell_ID	Location_Center_X	Location_Center_Y	Outline
```

Comment:
looks like this file contains a third column named `Outline`. Why is this file different than the others? Oh, is it b/c it is cytoplasm?

Reply:
Yes cytoplasm will contain the outlines.